lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -55,6 +55,10 @@
   <!-- PASS(responsive): image is used at full size -->
   <img src="lighthouse-unoptimized.jpg">
 
+  <!-- FAIL(optimized): image is not optimized -->
+  <!-- PASS(responsive): image is used at full size -->
+  <img src="https://github.com/GoogleChrome/lighthouse/raw/751a84547bfff97d1f6c4533d1be2266908dfdc9/assets/lighthouse-logo_no-light.png">
```

Comment:
any way we can do this without a network connection? It's nice to be able to run `npm run smoke` on an airplane :)